tests/scripts/fetch_manifests.sh

```diff
@@ -2,14 +2,25 @@
 
 set -x
 
+# Initial manifest directory
 DIR_NAME="/tmp/manifests/bootstrap-before-pivot"
-
+DIR_NAME_AFTER_PIVOT="/tmp/manifests/target-after-pivot"
+DIR_NAME_AFTER_REPIVOT="/tmp/manifests/bootstrap-after-repivot"
 # Check if manifest directory exists
-if [ -d "${DIR_NAME}" ]; then
-  DIR_NAME="/tmp/manifests/bootstrap-after-repivot"
+if [[ -d "${DIR_NAME}" && -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
+  DIR_NAME="${DIR_NAME_AFTER_REPIVOT}"
+  mkdir -p "${DIR_NAME}"
+  # Ephemeral cluster kubeconfig
+  kconfig="/home/metal3ci/.kube/config"
+elif [[ -d "${DIR_NAME}" && ! -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```

Comment:
`&&` goes outside the `[[ .. ]]`

```suggestion
elif [[ -d "${DIR_NAME}" ]] && [[ ! -d "${DIR_NAME_AFTER_PIVOT}" ]]; then
```